ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/queries/BlockchainQueries.java

```diff
@@ -75,6 +75,10 @@ public long headBlockNumber() {
     return blockchain.getChainHeadBlockNumber();
   }
 
+  public Optional<Long> fastSyncHeadBlockNumber() {
```

Comment:
I don't think this is used anywhere?  We should probably have a separate ticket for handling json-rpc requests - a bunch of those will need to be modified to handle fast sync.

Reply:
True. Was used in a hacked-in JSON-RPC method I added initially so I could see what was happening.  Removed.